Modules/DataModel/Sources/PocketCastsDataModel/Public/DataManager.swift

```diff
@@ -35,14 +35,14 @@ public class DataManager {
         DataManager.ensureDbFolderExists()
 
         let flags = SQLITE_OPEN_READWRITE | SQLITE_OPEN_CREATE | SQLITE_OPEN_FILEPROTECTION_NONE
-        let dbQueue = FMDatabaseQueue(path: DataManager.pathToDb(), flags: flags)!
+        let dbQueue = FMDBQueue(fmdbQueue: FMDatabaseQueue(path: DataManager.pathToDb(), flags: flags)!)
```

Comment:
This is the only place where code was changed, to wrap FMDB into FMDatabaseQueue.